`build-fqns.xsl` terminates parse instead of reporting error when `@` has no `φ` in scope

## What happens

`build-fqns.xsl` (`eo-parser/src/main/resources/org/eolang/parser/parse/build-fqns.xsl:83-89`) aborts the entire parse with a raw Saxon termination when `@` (`φ`) is referenced but no `φ` is in scope:

```xml
<xsl:if test="$find='φ'">
  <xsl:message terminate="yes">
    <xsl:text>The </xsl:text>
    <xsl:value-of select="$find"/>
    <xsl:text> object is used, but absent in self or parents scope</xsl:text>
  </xsl:message>
</xsl:if>
```

`terminate="yes"` stops the whole XSL pipeline, which surfaces to the caller as:

```
IllegalArgumentException: Shift 'com.yegor256.xsline.StClasspath@..' failed
Caused by: net.sf.saxon.expr.instruct.TerminationException: Processing terminated by xsl:message at line 85 in build-fqns.xsl
```

Verified by parsing a minimal source that uses `@` where no `φ` is in scope:
```
# Sample.

[] > sample
  @ > x
```

The user sees a stack trace naming an internal XSL file and a line number inside it, with no reference to their own source file, line, or position, and no `<errors>` XMIR entry at all — unlike every other diagnostic in this pipeline.

## What should happen

This should emit an `<errors><error line=".." pos="..">` entry (e.g. "`@` is used but no `φ` exists in this or any enclosing scope"), consistent with how every other parse-time validation in this pipeline reports problems, instead of terminating the whole XSL train.

## Steps to reproduce

1. Build `eo-parser`.
2. Parse a source that references `@` in a scope with no enclosing `φ`, e.g.:
   ```
   [] > sample
     @ > x
   ```
3. Observe the parse throws a raw `TerminationException` wrapped in `IllegalArgumentException`, naming `build-fqns.xsl` and an internal line number, with no `<errors>` entry and no reference to the user's own source location.

--- a/eo-parser/src/main/resources/org/eolang/parser/parse/build-fqns.xsl
+++ b/eo-parser/src/main/resources/org/eolang/parser/parse/build-fqns.xsl
@@ -82,13 +82,6 @@
         </xsl:apply-templates>
       </xsl:when>
       <xsl:otherwise>
-        <xsl:if test="$find='φ'">
-          <xsl:message terminate="yes">
-            <xsl:text>The </xsl:text>
-            <xsl:value-of select="$find"/>
-            <xsl:text> object is used, but absent in self or parents scope</xsl:text>
-          </xsl:message>
-        </xsl:if>
         <xsl:copy>
           <xsl:apply-templates select="node()|@*"/>
         </xsl:copy>
@@ -216,6 +209,38 @@
   <xsl:template match="node()|@*" mode="#all">
     <xsl:copy>
       <xsl:apply-templates select="node()|@*"/>
+    </xsl:copy>
+  </xsl:template>
+  <!--
+  A "φ" reference left unresolved by "with-package" above stays as a
+  literal "<o base='φ'>" in the transformed tree (nothing rewrites it).
+  Reporting this here, after the transform, rather than terminating the
+  whole XSL train mid-resolution (as this file used to), lets it surface
+  as a normal <errors> entry with the offending line, consistent with
+  every other diagnostic in this pipeline (see #6042).
+  -->
+  <xsl:template match="/object">
+    <xsl:variable name="transformed" as="item()*">
+      <xsl:apply-templates select="(node() except errors)|@*"/>
+    </xsl:variable>
+    <xsl:copy>
+      <xsl:sequence select="$transformed"/>
+      <xsl:variable name="errors" as="element()*">
+        <xsl:for-each select="$transformed//o[@base='φ']">
+          <error>
+            <xsl:attribute name="check" select="'build-fqns'"/>
+            <xsl:attribute name="line" select="if (@line) then @line else 0"/>
+            <xsl:attribute name="severity" select="'error'"/>
+            <xsl:text>The φ object is used, but absent in self or parents scope</xsl:text>
+          </error>
+        </xsl:for-each>
+      </xsl:variable>
+      <xsl:if test="not(empty($errors)) or exists(/object/errors)">
+        <errors>
+          <xsl:apply-templates select="/object/errors/error"/>
+          <xsl:copy-of select="$errors"/>
+        </errors>
+      </xsl:if>
     </xsl:copy>
   </xsl:template>
 </xsl:stylesheet>

--- a/eo-parser/src/test/resources/org/eolang/parser/eo-syntax/unresolved-phi-is-rejected.yaml
+++ b/eo-parser/src/test/resources/org/eolang/parser/eo-syntax/unresolved-phi-is-rejected.yaml
@@ -1,0 +1,10 @@
+# SPDX-FileCopyrightText: Copyright (c) 2016-2026 Objectionary.com
+# SPDX-License-Identifier: MIT
+---
+# yamllint disable rule:line-length
+sheets: []
+asserts:
+  - /object/errors/error[contains(text(),'φ')]
+input: |
+  [] > sample
+    @ > x

--- a/eo-parser/src/test/resources/org/eolang/parser/eo-syntax/unresolved-phi-is-rejected.yaml
+++ b/eo-parser/src/test/resources/org/eolang/parser/eo-syntax/unresolved-phi-is-rejected.yaml
@@ -4,6 +4,9 @@
 # yamllint disable rule:line-length
 sheets: []
 asserts:
+  - /object/errors/error[@check='build-fqns']
+  - /object/errors/error[@severity='error']
+  - /object/errors/error[@line='2']
   - /object/errors/error[contains(text(),'φ')]
 input: |
   [] > sample

